config.js

```diff
@@ -0,0 +1,6 @@
+var SomaPlayerConfig = {
```

Comment:
This file could be deleted. config.js.example is in the root directory, but config.js should end up in extension/.